train.py

```diff
@@ -12,40 +20,36 @@
 from wavenet import WaveNet
 
 BATCH_SIZE = 1
-CHANNELS = 256
 DATA_DIRECTORY = './VCTK-Corpus'
-FILTER_WIDTH = 2
 LOGDIR = './logdir'
 NUM_STEPS = 1000
 LEARNING_RATE = 0.10
+NETWORK = './layout.json'
```

Comment:
I think `NETWORK_CONFIG` or just `CONFIG` would be a better name, as we're not storing weights in the file.